apps/io-services-cms-webapp/src/config.ts

```diff
@@ -32,12 +32,30 @@ export const JiraConfig = t.interface({
 });
 export type JiraConfig = t.TypeOf<typeof JiraConfig>;
 
-// global app configuration
+// Service Principal Creds configuration
+export const ServicePrincipalConfig = t.interface({
```

Comment:
Naming: _ServicePrincipal_ of what? Also: we can use comments to explain **why** we write something, just repeating the code in plain English has usually little value.

Reply:
FYI https://github.com/pagopa/developer-portal/pull/46#discussion_r1179390139